content/en/blog/2023/announcing-notation-rc4.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Notation v1.0.0-RC.4 is available!
+author:  "Notation Release Team"
+date: 2023-04-21
+draft: false
+---
+
+The Notation maintainers are pleased to announce the releases of Notation CLI v1.0.0-rc.4, notation-go v1.0.0-rc.4, and notation-core-go v1.0.0-rc.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release adds the following significant enhancements:
+
+- Validate Certificate revocation with OCSP
+- Support experimental features
+- Introduce new CLI command `notation policy` for managing trust policy configuration
+- Support OCI distribution referrers API
+- Introduce experimental feature of signing and verifying images as OCI image layout
+
+### Validate Certificate revocation with OCSP
+
+OCSP stands for Online Certificate Status Protocol. It is a protocol used to check the revocation status of digital certificates. When a certificate is revoked, it means that it is no longer valid and should not be trusted. To enforce validating certificate revocation, the verification level of trust policy configuration should be set to `strict` as following. Once this is done, the CLI command `notation verify` can be used to validate the signatures including checking certificate revocation status with OCSP.
+
+```json
+"signatureVerification": {
+    "level" : "strict" 
+}
+```
+
+### Support experimental features
```

Comment:
Same as above

Reply:
Need a short version for title, any other proposal?